sdlib/d/gc/tcache.d

```diff
@@ -492,6 +492,25 @@ private:
 		triggerDeallocationEvent(npages * PageSize);
 	}
 
+	/**
+	 * Free directly to the arena, skip all threadCache metadata/caches.
+	 * Safe to call from a signal handler.
+	 */
+	void freeToArena(void* ptr) {
+		import d.gc.emap, d.gc.base;
+		auto emap = CachedExtentMap(&gExtentMap, &gBase);
```

Comment:
If this lives in the thread cache, this should sue the emap and base of the thread cache. But this probably shouldn't live in there anyways.

Reply:
Was moved.